BLL/PaginatedList.cs

```diff
@@ -6,42 +6,29 @@ namespace BLL
 {
     public class PaginatedList<T> : List<T>
     {
-        public int PageIndex { get; private set; }
-        public int TotalPages { get; private set; }
+        public int PageIndex { get; }
+        public int TotalPages { get; }
 
         public PaginatedList(List<T> items, int count, int pageIndex, int pageSize)
         {
             PageIndex = pageIndex;
             TotalPages = (int)Math.Ceiling(count / (double)pageSize);
 
-            this.AddRange(items);
+            AddRange(items);
         }
 
-        public bool HasPreviousPage
-        {
-            get
-            {
-                return (PageIndex > 1);
-            }
-        }
+        public bool HasPreviousPage => (PageIndex > 1);
+
+        public bool HasNextPage => (PageIndex < TotalPages);
 
-        public bool HasNextPage
-        {
-            get
-            {
-                return (PageIndex < TotalPages);
-            }
-        }
         public string IsOnCurrentPage(int pageIndex, int currentPageIndex)
         {
             if (pageIndex == currentPageIndex)
             {
                 return "disabled";
             }
-            else
-            {
-                return "";
-            }
```

Comment:
While those changes are okay, I don't know why this is done on adding review branch.

Reply:
Yes, I know... Sorry... I just wanted to leave code better. ;)